### ISSUE TITLE
Installation fails under Python 3.5 on Windows

### ISSUE THREAD

**Issue by dswhite42** [scrapinghub/dateparser#128]
Installing dateparser under Windows Python 3.5 does not work:

    C:\Python35\Scripts> .\pip install --upgrade
        c:\users\dswhite42\Downloads\dateparser-0.3.1-py2.py3-none-any.whl

    Processing c:\users\dswhite42\downloads\dateparser-0.3.1-py2.py3-none-any.whl
    Collecting jdatetime (from dateparser==0.3.1)
      Using cached jdatetime-1.7.1.tar.gz
    Collecting python-dateutil>=2.3 (from dateparser==0.3.1)
      Using cached python_dateutil-2.4.2-py2.py3-none-any.whl
    Collecting PyYAML (from dateparser==0.3.1)
      Using cached PyYAML-3.11.tar.gz
    Requirement already up-to-date: six>=1.5 in c:\python35\lib\site-packages
      (from python-dateutil>=2.3->dateparser==0.3.1)
    Installing collected packages: jdatetime, python-dateutil, PyYAML,
      dateparser
      Running setup.py install for jdatetime
      Running setup.py install for PyYAML

        Complete output from command c:\python35\python.exe -c "import
    setuptools,tokenize;__file__='C:\\Users\\dswhite42\\AppData\\Local\\Temp\\
    pip-build-xkws71jn\\PyYAML\\setup.py';exec(compile(getattr(tokenize,
    'open',open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))"
    install--record
    C:\Users\dswhite42\AppData\Local\Temp\pip-oim5fb3l-record\install-record.txt
    --single-version-externally-managed --compile:
        running install
        running build  
        running build_py
        creating build  
        creating build\lib.win-amd64-3.5
        creating build\lib.win-amd64-3.5\yaml
        copying lib3\yaml\composer.py -> build\lib.win-amd64-3.5\yaml
        copying lib3\yaml\constructor.py -> build\lib.win-amd64-3.5\yaml
        copying lib3\yaml\cyaml.py -> build\lib.win-amd64-3.5\yaml
        copying lib3\yaml\dumper.py -> build\lib.win-amd64-3.5\yaml
        copying lib3\yaml\emitter.py -> build\lib.win-amd64-3.5\yaml
        copying lib3\yaml\error.py -> build\lib.win-amd64-3.5\yaml  
        copying lib3\yaml\events.py -> build\lib.win-amd64-3.5\yaml
        copying lib3\yaml\loader.py -> build\lib.win-amd64-3.5\yaml
        copying lib3\yaml\nodes.py -> build\lib.win-amd64-3.5\yaml 
        copying lib3\yaml\parser.py -> build\lib.win-amd64-3.5\yaml  
        copying lib3\yaml\reader.py -> build\lib.win-amd64-3.5\yaml 
        copying lib3\yaml\representer.py -> build\lib.win-amd64-3.5\yaml
        copying lib3\yaml\resolver.py -> build\lib.win-amd64-3.5\yaml   
        copying lib3\yaml\scanner.py -> build\lib.win-amd64-3.5\yaml 
        copying lib3\yaml\serializer.py -> build\lib.win-amd64-3.5\yaml
        copying lib3\yaml\tokens.py -> build\lib.win-amd64-3.5\yaml
        copying lib3\yaml\__init__.py -> build\lib.win-amd64-3.5\yaml
        running build_ext
        creating build\temp.win-amd64-3.5
        creating build\temp.win-amd64-3.5\Release
        checking if libyaml is compilable
        error: [WinError 2] The system cannot find the file specified
    
        ----------------------------------------
    Command "c:\python35\python.exe -c "import setuptools,
    tokenize;__file__='C:\\Users\\dswhite42\\AppData\\Local\\Temp\\pip-build-xkws71jn\\PyYAML\\s
    etup.py';exec(compile(getattr(tokenize, 'open',
    open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install
    --record C:\Users\dswhite42\AppData\Local\Temp\pip-oim5fb3l-record\install-record.txt
    --single-version-externally-managed --compile" failed with error code 1 in
    C:\Users\ dswhite42\AppData\Local\Temp\pip-build-xkws71jn\PyYAML

Admittedly, this is probably a PyYAML problem, since the library has not been updated in well over a year and thus doesn't have a precompiled binary for Python 3.5 .  Given that dateparser has a dependency on it, though, this will probably be an ongoing problem.

Thanks.

**Comment by asadurski** [comment]
Thank you for reporting this, @dswhite42. I confirm, I was able to reproduce the problem.
The quick workaround would be to build PyYAML from downloaded source with `python setup.py install`, prior to installing `dateparser`. We are looking for more convenient solutions though.

**Comment by asadurski** [comment]
I will just repost your own finding (unofficial Windows binaries for Python 3.5), if anyone else has the same problem:
http://www.lfd.uci.edu/~gohlke/pythonlibs/#pyyaml

Source: https://bitbucket.org/xi/pyyaml/issues/51/binary-windows-installers-wheels-for

**Comment by RichardLangFromNZ** [comment]
Had same issue trying to install PyYAML into Python 3.5 install with pip3, but couldn't even get a manual download and install to work...

```
C:\Temp\PyYAML-3.11>py -3 setup.py install
running install
running build
running build_py
creating build
creating build\lib.win32-3.5
creating build\lib.win32-3.5\yaml
copying lib3\yaml\composer.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\constructor.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\cyaml.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\dumper.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\emitter.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\error.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\events.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\loader.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\nodes.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\parser.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\reader.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\representer.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\resolver.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\scanner.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\serializer.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\tokens.py -> build\lib.win32-3.5\yaml
copying lib3\yaml\__init__.py -> build\lib.win32-3.5\yaml
running build_ext
creating build\temp.win32-3.5
creating build\temp.win32-3.5\Release
checking if libyaml is compilable
error: [WinError 2] The system cannot find the file specified
```

**Comment by waqasshabbir** [comment]
Fixed in PR #190
